frappe/public/js/frappe/model/model.js

```diff
@@ -650,6 +650,28 @@ $.extend(frappe.model, {
 		}
 	},
 
+	round_floats_childs_in: function(arr, fieldnames) {
```

Comment:
name of the function is very unintuitive... can we have better naming for this function?

Reply:
what do you suggest me? I just used a similar name to the original function "round_floats_in"